tutorials/Quantile_Interval_And_Interval_Scores.ipynb

```diff
@@ -536,21 +547,21 @@
     }
    ],
    "source": [
-    "overal_qis_sc1 = quantile_interval_score(\n",
+    "overall_qis_sc1 = quantile_interval_score(\n",
     "    obs=obs_da,\n",
     "    fcst_lower_qtile=forecast_lower_da_1,\n",
     "    fcst_upper_qtile=forecast_upper_da_1,\n",
     "    lower_qtile_level=0.1,\n",
     "    upper_qtile_level=0.6\n",
     ")\n",
-    "overal_qis_sc1"
+    "overall_qis_sc1"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As mentioned, we can use both the *quantile interval score* and the *interval score* for Scenario 2 due the symmetric quantile range of forecast intervals in this scenario. Both scores will yield the same results. Let's verify this now:"
+    "As mentioned, we can use either the *quantile interval score* or the *interval score* for Scenario 2 due to the symmetric quantile range of forecast intervals in this scenario. Both scores will yield the same results. Let's verify this now:"
```

Comment:
Quantile interval score and interval score are both in italics here. I think this may be an oversight?